bundles/org.openhab.binding.goecharger/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="goecharger"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="goe">
+		<label>Go-eCharger</label>
+		<description>Go-eCharger thing that represents the wallbox configuration and readings</description>
+
+		<channels>
+			<channel id="maxCurrent" typeId="current" />
+			<channel id="pwmSignal" typeId="pwm" />
+			<channel id="error" typeId="err" />
+			<channel id="voltageL1" typeId="vl1" />
+			<channel id="voltageL2" typeId="vl2" />
+			<channel id="voltageL3" typeId="vl3" />
+			<channel id="currentL1" typeId="cl1" />
+			<channel id="currentL2" typeId="cl2" />
+			<channel id="currentL3" typeId="cl3" />
+			<channel id="powerL1" typeId="pl1" />
+			<channel id="powerL2" typeId="pl2" />
+			<channel id="powerL3" typeId="pl3" />
+			<channel id="phases" typeId="pha" />
+			<channel id="sessionChargeEnergyLimit" typeId="scl" />
+			<channel id="sessionChargedEnergy" typeId="scs" />
+			<channel id="totalChargedEnergy" typeId="eto" />
+			<channel id="allowCharging" typeId="alw" />
+			<channel id="cableCurrent" typeId="cbl" />
+			<channel id="temperature" typeId="tmp" />
+			<channel id="firmware" typeId="fmw" />
+			<channel id="accessState" typeId="ast" />
+		</channels>
+
+		<config-description>
+			<parameter name="ip" type="text" required="true">
+				<label>IP Address</label>
+				<description>The IP address of the Go-eCharger</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" required="false" unit="s">
+				<label>Refresh Interval</label>
+				<description>Refresh interval for acquiring data from Go-eCharger in seconds</description>
+				<unitLabel>s</unitLabel>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="ast">
+		<item-type>String</item-type>
+		<label>Access State</label>
+		<description>Current access state of the go-eCharge</description>
```

Comment:
Since all String states are known, you can add state options for these. That makes them translatable and it allows them to show up nicer in UIs:

```xml
		<state readOnly="true">
			<options>
				<option value="OPEN">Open</option>
				<option value="RFID">RFID</option>
				<option value="TIMER">Timer</option>
				<option value="AWATTAR">aWATTar</option>
				<option value="UNKNOWN">Unknown</option>
			</options>
		</state>
```

See also: https://www.openhab.org/docs/developer/bindings/thing-xml.html#state-description